src/knockout-jqAutocomplete.js

```diff
@@ -94,9 +94,16 @@
         //the binding's update function. keep value in sync with model
         this.update = function(element, valueAccessor) {
             var options = unwrap(valueAccessor()),
-                value = unwrap(options && options.value);
+                value = unwrap(options && options.value),
+                inputProp = options.inputProp || options.labelProp || options.valueProp,
+                valueProp = options.valueProp,
+                sources = ko.unwrap(options.source);
 
-            element.value = value;
+            if (sources.constructor === Array) {
```

Comment:
Since, we have jQuery, I would prefer to use `$.isArray`. They do it a little differently and will use the array's native `isArray` function if it is there. Although this would surely be fine 99.9% of the time.